openedge.properties

```diff
@@ -18,3 +18,9 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.19.2.date=2023-01-10
```

Comment:
This version can't be left compatible with `LATEST`; only one version can be marked compatible with `LATEST` at a time. Set the end of this version's range to `10.3` (or `10.3.*`)

Reply:
Changed to 9.9.* (this version was not tested with 10.x)